Crashlytics/Crashlytics/Handlers/FIRCLSSignal.c

```diff
@@ -179,17 +179,17 @@ bool FIRCLSSignalSafeInstallPreexistingHandlers(FIRCLSSignalReadContext *roConte
 #endif
 
   // re-install the original handlers, if any
-  success = true;
-  for (int i = 0; i < FIRCLSSignalCount; ++i) {
-    if (roContext->originalActions[i].sa_sigaction == NULL) {
-      continue;
+  FIRCLSSignalEnumerateHandledSignals(^(int idx, int currentSignal) {
+    success = true;
```

Comment:
Worried this'll overwrite a `success = false` from a previous failed iteration - should we just set it to true in the `__block bool success;`?